tables/gurumuki.cti

```diff
@@ -1,7 +1,13 @@
 # This table contains braille codes and rules for Gurumukhi script.
-#
-# Copyright (C) 2014 National Institute for Visually Handicapped, 116, Rajpur Road, Dehradun, Uttrakhand, India
-#
+
+#-author-name: Dipendra Manocha
+#-author-email: dmanocha@daisy.org
```

Comment:
Shouldn't your name be in this list as well? There can be multiple `#-author-name` lines.